bigframes/ml/utils.py

```diff
@@ -55,44 +63,16 @@ def _convert_to_series(frame: ArrayType) -> bpd.Series:
         return typing.cast(bpd.Series, frame[label])
     if isinstance(frame, bpd.Series):
         return frame
+    if isinstance(frame, pd.DataFrame):
+        # Recursively call this method to re-use the length-checking logic
+        return _convert_to_series(bpd.read_pandas(frame))
```

Comment:
we might not always want the default session, if the other argument is a bigframes object with a non-default session. the core version uses the session from the co-argument

Reply:
Good point! I decided to use the sessions provided from the bqml_model whenever possible. The global session acts as a default.